app/AppKernel.php

```diff
@@ -23,6 +23,7 @@ public function registerBundles()
             new Knp\Bundle\PaginatorBundle\KnpPaginatorBundle(),
             new CodeExplorerBundle\CodeExplorerBundle(),
             new AppBundle\AppBundle(),
+            new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
```

Comment:
This is a demo app, and it has no initial database migration step with e.g. a default admin login - it uses the fixtures bundle instead, so it's fine to use that in production in this special case.